_security/access-control/users-roles.md

```diff
@@ -138,6 +138,64 @@ See [YAML files]({{site.url}}{{site.baseurl}}/security/configuration/yaml/#roles
 
 See [Create role mapping]({{site.url}}{{site.baseurl}}/security/access-control/api/#create-role-mapping).
 
+## Read-only roles
+
+As with any role in OpenSearch, read-only roles can be configured using the following three methods: 
+- Modifying the `yml` configuration files.
+- Using the Cluster Settings API. 
+- Using OpenSearch Dashboards. 
+
+The most user friendly approach for anyone getting familiar with Roles and role mappings is to use OpenSearch Dashboards, as it is easy to navigate the creation of roles and assign those roles to users. The basic steps of creating roles, mapping, and users can be found in the [User and roles documentation](https://opensearch.org/docs/latest/security/access-control/users-roles/).
+
+### Creating a basic read-only role
+
+
+To create a basic read-only role which allows access to OpenSearch Dashboards, view existing dashboards, visualizations and query different indexes, use one the following permissions. 
```

Comment:
I'm assuming that in the **Create user** view in Dashboards that we would use all of these permissions for the `read_only` role. Is that correct?

```suggestion
To create a basic read-only role which allows access to OpenSearch Dashboards, view existing dashboards, visualizations and query different indexes, use the following permissions.
```